src/content/api/events.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Events
+description: Events and related endpoints
+nav:
+  path: API/Events
+  order: 1
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+An event is an activity from an action which subsequently undergoes processing by job workers.
```

Comment:
Does an integrator know what a job worker is?

Reply:
perhaps something like this:

An event is an activity triggered by an action. Some events are native to Centrapay, these native events are handled automatically, requiring no integrator involvement.